app/src/main/java/protect/card_locker/LoyaltyCardViewActivity.java

```diff
@@ -428,6 +429,8 @@ public void onStopTrackingTouch(SeekBar seekBar) {
         maximizeButton.setOnClickListener(v -> setFullscreen(true));
         minimizeButton.setOnClickListener(v -> setFullscreen(false));
 
+        DBHelper.updateLoyaltyCardUsageNumber(database, loyaltyCardId);
+
```

Comment:
Would be nice to put this either near the top or all the way at the bottom for clarity.

Reply:
Hi, i don't know where to put this function, i think it going to be null :/